.github/workflows/tag.yaml

```diff
@@ -0,0 +1,69 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Get TAG
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      - name: build
+        run: make build
+      - name: get tools
+        run: ./contrib/get_tools.sh
+      - name: lint chart
+        run: ./contrib/lint_chart.sh
+      - name: create cluster
+        run: ./contrib/create_cluster.sh
+      - name: deploy deploy
+        run: OPERATOR=0 ./contrib/deploy.sh
+      - name: run function
+        run: ./contrib/run_function.sh
+      - name: stop dev cluster
+        run: ./contrib/stop_dev.sh
+      - name: Sleep for 10 seconds
```

Comment:
Can you do an inline GitHub script instead if a wait is needed? Third-party actions are disabled.

Reply:
https://github.com/actions/github-script